integration-tests/src/test/resources/sitconfig/scripts/create-domain-auto-custom-sit-config20.py

```diff
@@ -0,0 +1,254 @@
+set('ListenPort', admin_port)
+set('Name', admin_server_name)
+
+create('T3Channel', 'NetworkAccessPoint')
+cd('/Servers/%s/NetworkAccessPoints/T3Channel' % admin_server_name)
+set('PublicPort', t3_channel_port)
+set('PublicAddress', 'junkvalue')
+# Dont set listenaddress, introspector overrides automatically with sit-config
+#set('ListenAddress', '%s-%s' % (domain_uid, admin_server_name_svc))
+set('ListenPort', t3_channel_port)
+
+cd('/Servers/%s' % admin_server_name)
+create(admin_server_name, 'Log')
+cd('/Servers/%s/Log/%s' % (admin_server_name, admin_server_name))
+# Give incorrect filelog, introspector overrides with sit-config
+set('FileName', 'dirdoesnotexist')
+
+
+
+# Set the admin user's username and password
+# ==========================================
+cd('/Security/%s/User/weblogic' % domain_name)
+cmo.setName(admin_username)
+cmo.setPassword(admin_password)
+
+# Write the domain and close the domain template
+# ==============================================
+setOption('OverwriteDomain', 'true')
+
+# Create a cluster
+# ======================
+cd('/')
+cl = create(cluster_name, 'Cluster')
+
+if cluster_type == "CONFIGURED":
+
+    # Create managed servers
+    for index in range(0, number_of_ms):
+        cd('/')
+
+        msIndex = index + 1
+        name = '%s%s' % (managed_server_name_base, msIndex)
+        name_svc = '%s%s' % (managed_server_name_base_svc, msIndex)
+
+        create(name, 'Server')
+        cd('/Servers/%s/' % name)
+        print('managed server name is %s' % name);
+        set('ListenAddress', '%s-%s' % (domain_uid, name_svc))
+        set('ListenPort', server_port)
+        set('NumOfRetriesBeforeMSIMode', 0)
+        set('RetryIntervalBeforeMSIMode', 1)
+        set('Cluster', cluster_name)
+
+else:
+    print('Configuring Dynamic Cluster %s' % cluster_name)
+
+    templateName = cluster_name + "-template"
+    print('Creating Server Template: %s' % templateName)
+    st1 = create(templateName, 'ServerTemplate')
+    print('Done creating Server Template: %s' % templateName)
+    cd('/ServerTemplates/%s' % templateName)
+    cmo.setListenPort(server_port)
+    cmo.setListenAddress('%s-%s${id}' % (domain_uid, managed_server_name_base_svc))
+    cmo.setCluster(cl)
+    print('Done setting attributes for Server Template: %s' % templateName);
+
+
+    cd('/Clusters/%s' % cluster_name)
+    create(cluster_name, 'DynamicServers')
+    cd('DynamicServers/%s' % cluster_name)
+    set('ServerTemplate', st1)
+    set('ServerNamePrefix', managed_server_name_base)
+    set('DynamicClusterSize', number_of_ms)
+    set('MaxDynamicClusterSize', number_of_ms)
+    set('CalculatedListenPorts', false)
+    # set('Id', 1)
+
+    print('Done setting attributes for Dynamic Cluster: %s' % cluster_name);
+
+createDataSource('JdbcTestDataSource-0', 'jdbc/JdbcTestDataSource-0', 'jdbc:oracle:thin:@//slcai724.us.oracle.com:1583/w03ys12c', 'j2ee', 'j2ee', admin_server_name)
```

Comment:
Internal URL

Reply:
will replace it with mysql

Reply:
SECURITY VIOLATION
do not check oracle internal server names into public repositories

Reply:
Fixed with mysql db container created during test run.